sky/clouds/kubernetes.py

```diff
@@ -110,9 +110,12 @@ def _unsupported_features_for_resources(
             # Controllers cannot spin up new pods with exec auth.
             unsupported_features[
                 clouds.CloudImplementationFeatures.HOST_CONTROLLERS] = message
-            # Pod does not have permissions to terminate itself with exec auth.
+            # Pod does not have permissions to down itself with exec auth.
             unsupported_features[
-                clouds.CloudImplementationFeatures.AUTO_TERMINATE] = message
+                clouds.CloudImplementationFeatures.AUTODOWN] = message
+            # Pod does not have permissions to stop itself with exec auth.
+            unsupported_features[
+                clouds.CloudImplementationFeatures.AUTOSTOP] = message
```

Comment:
Auto stop is not supported regardless of whether exec based auth is used:
```suggestion
        # Pod does not have permissions to stop itself with exec auth.
        unsupported_features[clouds.CloudImplementationFeatures.AUTOSTOP] = 'Auto-stop is not supported on Kubernetes.'
```